doc/DOCUMENTATION.org

```diff
@@ -1676,34 +1678,32 @@ selected layout.
-| ~o~          | open a custom layout                                       |
-| ~p~          | previous layout in list                                    |
-| ~r~          | remove current buffer from layout                          |
-| ~R~          | rename current layout                                      |
-| ~s~          | save layouts                                               |
-| ~t~          | display a buffer without adding it to the current layout   |
-| ~w~          | workspaces transient state (needs eyebrowse layer enabled) |
-| ~x~          | kill current layout with its buffers                       |
-| ~X~          | kill other layouts with their buffers                      |
+| Key Binding         | Description                                                |
+|---------------------+------------------------------------------------------------|
+| ~SPC l~             | initiate transient state                                   |
+| ~?~                 | toggle the documentation                                   |
+| ~[0..9]~            | switch to nth layout                                       |
+| ~[C-0..C-9]~        | switch to nth layout and keep the transient state active   |
+| ~<tab>~             | switch to the latest layout                                |
+| ~a~                 | add a buffer to the current layout                         |
+| ~A~                 | add all the buffers from another layout in the current one |
+| ~b~                 | select a buffer in the current layout                      |
+| ~d~                 | delete the current layout and keep its buffers             |
+| ~D~                 | delete the other layouts and keep their buffers            |
+| ~h~                 | go to default layout                                       |
+| ~l~                 | select/create a layout                                     |
+| ~L~                 | load layouts from file                                     |
+| ~n~ or ~C-l~        | next layout in list                                        |
+| ~N~ or ~p~ or ~C-h~ | previous layout in list                                    |
+| ~o~                 | open a custom layout                                       |
+| ~r~                 | remove current buffer from layout                          |
+| ~R~                 | rename current layout                                      |
+| ~s~                 | save layouts                                               |
+| ~t~                 | display a buffer without adding it to the current layout   |
+| ~w~                 | workspaces transient state (needs eyebrowse layer enabled) |
+| ~x~                 | kill current layout with its buffers                       |
+| ~X~                 | kill other layouts with their buffers                      |
+| Any other key       | leave the layout transient state                           |
```

Comment:
Instead of adding this line to all the transient state keybinding description I would prefer if we removed it from the existing ones. This functionality should be implied by the fact that it's a transient state.

Reply:
Do you mean the line
`| Any other key       | leave the layout transient state                           | `?
 It might be a small technicality but isn't it possible to have transient states which don't exit on any other key, by setting the `:foreign-keys` prop? (I'm not sure if there are any in the default Spacemacs layers)

Reply:
Ok, edited the rest of the doc to remove those references. Let me know if/how I should squash the commits :)